src/traces/scatter/plot.js

```diff
@@ -419,9 +419,6 @@ function plotOne(gd, idx, plotinfo, cdscatter, cdscatterAll, element, transition
         var enter = join.enter().append('path')
             .classed('point', true);
 
-        enter.call(Drawing.pointStyle, trace)
```

Comment:
> I'm still think that translatePoints and pointStyle should be applied to the merge

That's always been the case [here](https://github.com/plotly/plotly.js/blob/cea9455ba353fe62ef374454cc954c4b7945bb0a/src/traces/scatter/plot.js#L436) and [here](https://github.com/plotly/plotly.js/blob/cea9455ba353fe62ef374454cc954c4b7945bb0a/src/traces/scatter/plot.js#L439).